packages/decent.js/docs/client.md

```diff
@@ -0,0 +1,62 @@
+# const Client = require('decent.js')
+Constructor takes no arguments:
+
+```js
+const Client = require('decent.js')
```

Comment:
I feel like it would easier to read if this line were:
```js
const Decent = require('decent.js')
```

Having two variables of the same name but with different capitalization is bound to cause confusion.

Reply:
That's a really common pattern :thinking:

Reply:
Huh. Never seen it. :package: